src/transformers/modeling_distilbert.py

```diff
@@ -461,17 +464,14 @@ def forward(
 
         if inputs_embeds is None:
             inputs_embeds = self.embeddings(input_ids)  # (bs, seq_length, dim)
-        tfmr_output = self.transformer(
+        return self.transformer(
             x=inputs_embeds,
             attn_mask=attention_mask,
             head_mask=head_mask,
             output_attentions=output_attentions,
             output_hidden_states=output_hidden_states,
+            return_tuple=return_tuple,
         )
-        hidden_state = tfmr_output[0]
-        output = (hidden_state,) + tfmr_output[1:]
```

Comment:
Note that the previous code was doing nothing so we can directly return the result of `self.transformer`.